drivers/soundwire/intel.c

```diff
@@ -880,6 +1274,7 @@ static int sdw_master_read_intel_prop(struct sdw_bus *bus)
 				 "intel-sdw-ip-clock",
 				 &prop->mclk_freq);
 
+	prop->mclk_freq /= 2;
```

Comment:
Is it better to have commit to describe why ``/= 2`` is required?

Reply:
done